app/controllers/diary_entry_controller.rb

```diff
@@ -69,6 +80,30 @@ def comment
     render :action => "no_such_entry", :status => :not_found
   end
 
+  def subscribe
+    diary_entry = DiaryEntry.find(params[:id])
+
+    if ! diary_entry.subscribers.exists?(@user.id)
```

Comment:
rubocop has already complained about that, and quite a number of other things...